backend/errors/httperror_test.go

```diff
@@ -0,0 +1,64 @@
+package errors
+
+import (
+	"bytes"
+	"errors"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+)
+
+func TestHandleHTTPError(t *testing.T) {
+	tests := []struct {
+		testName       string
```

Comment:
Please rename this to `name` to reduce repetitiveness when accessing the corresponding value.

Style reference: https://google.github.io/styleguide/go/best-practices#avoid-repetition